hello_microverse.html

```diff
@@ -0,0 +1 @@
+<h1>Hello Microverse!</h1>
```

Comment:
- [ ] You're required to create .html which usualy have this generic markup. :point_down: 

```
   <html lang="en">

      <head>
         <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial- 
         scale=1.0">
        <link rel="stylesheet" href="style.css">
        <title>Document</title> 
     </head>
     <body>
         <h1 class="header">Hello Microverse</h1>
     </body>
  </html>
```

- [ ] You need to link your .css file with your .html file so that the styling could be effective. 
- [ ] You're required to give a class name to the header and use that to style the content of the header.